<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modificar</class>
 <widget class="QDialog" name="modificar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>590</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="mb_no">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>542</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_pago">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>336</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="mr_si">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>516</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Si</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_capacidad">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>402</y>
     <width>48</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>303</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Velocidad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>590</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_mesero">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>48</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="mb_si">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>514</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Si</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_velocidad">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>303</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="mr_no">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>544</y>
     <width>50</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_peso">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>369</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_descripcion">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>75</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>402</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Capacidad</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_kilometro">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>458</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>369</y>
     <width>33</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Peso</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>336</y>
     <width>35</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pago</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_habitacion">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>460</y>
     <width>48</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>491</y>
     <width>42</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ba√±os</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>75</y>
     <width>80</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Descripcion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>493</y>
     <width>84</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Restaurante</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>52</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Elegido</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_elegido">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>435</y>
     <width>75</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Kilometros</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>437</y>
     <width>74</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Habitacion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>437</y>
     <width>58</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Meseros</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>42</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ID&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_id">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>42</y>
     <width>146</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
